rotkehlchen/tests/conftest.py

```diff
@@ -158,10 +161,10 @@ def get_cassette_dir(request: pytest.FixtureRequest) -> Path:
     return Path(request.node.path).relative_to(TESTS_ROOT_DIR).with_suffix('')
 
 
-@pytest.fixture
-def vcr_config(request: pytest.FixtureRequest, default_cassette_name: str) -> dict[str, Any]:
+@pytest.fixture(scope='module')
+def vcr_config(_request: pytest.FixtureRequest) -> dict[str, Any]:
     """
-    vcrpy config
+    vcrpy configd
```

Comment:
I guess this is accidental here right?

Reply:
It's yabir's world-famous typos

Reply:
This smells like accidentally pressing a button before moving to another file :) 

This happens to me sometimes too 😛

Reply:
It was not accidental. The CI got stuck and I wanted to force push just to test it :see_no_evil:

Reply:
maybe rebase the branch and change the commit? 😛